Library/Formula/xhyve.rb

```diff
@@ -0,0 +1,18 @@
+class Xhyve < Formula
+  desc "xhyve, a lightweight OS X virtualization solution"
+  homepage "http://www.pagetable.com/?p=831"
+  url "https://github.com/mist64/xhyve/archive/84868be7a65305a3a7ab8f3c197d6df93ebbe5dc.tar.gz"
+  version "0.0.0"
+  sha256 "9d1618283bf35cc54f095782aaebd2796454a4a9c9e563f343e3a4093c7c9424"
+  head "https://github.com/mist64/xhyve.git"
+
+  def install
+    system "make"
+
+    bin.install "build/xhyve"
+  end
+
+  test do
+    system "xhyve -h"
```

Comment:
Please use `system "#{bin}/xhyve", "-h"` instead of `system "xhyve -h"`. Could you write a test that does something more substantial than e.g. `--help`/`--version`? Thanks!